tests/debug_printf_test/Makefile

```diff
@@ -1,19 +1,13 @@
-ifeq ($(USING_XWAF),1)
-all:
-	xwaf configure build
```

Comment:
The support for (x)waf has been removed in the Makefile. Does it mean that we shouldn't use waf when building these tests?
If this is the case, can we remove the wscript files in the tests?

Reply:
I removed it because it was a pattern that I haven't seen elsewhere - if you want to build with waf, it's clearer to call waf directly rather than call `xmake USING_XWAF=1`

The wscripts remain because waf is part of the tools and gives us benefit in terms of build speed.